Azure-Samples/communication-services-web-calling-hero
# Introducing Remote streams selection/limit

## Purpose

* SDK only support 1 remote stream currently. Introducing new feature that allows media gallery to only render 1 stream at a time base on a criteria, should be able to dial up the number of streams easily once SDK supports more.
For more information about the criteria please checkout this doc https://microsoft-my.sharepoint-df.com/:w:/p/abelsaba/EeT7O7HPGuVDji-hnHN77RYBqmbFuprTIbIW2tmERNEf_g?e=ryE0t6 

## Does this introduce a breaking change?

```
[x] Yes
[] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
After running the sample code locally, test by creating group meeting with multiple clients, turn on/off video and mute/unmute remote participants and notice the behavior of the media gallery, which streams are being rendered
```

## What to Check
Verify that the following are valid
* Edge cases
* Feedback regarding the criteria

## Other Information
